revista-nordestina-de-biologia.csl

```diff
@@ -0,0 +1,437 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
+  <info>
+    <title>Revista Nordestina de Biologia</title>
+    <title-short>Rev. Nordestina Biol.</title-short>
+    <id>http://www.zotero.org/styles/revista-nordestina-de-biologia</id>
+    <link rel="self" href="http://www.zotero.org/styles/revista-nordestina-de-biologia"/>
```

Comment:
Please add the template style in here. See our requirements documentation: https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md